src/plugins/kibana_react/public/page_template/page_template.tsx

```diff
@@ -92,8 +93,8 @@ export const KibanaPageTemplate: FunctionComponent<KibanaPageTemplateProps> = ({
       pageHeader={pageHeader}
       pageSideBar={pageSideBar}
       pageSideBarProps={{
-        ...rest.pageSideBarProps,
-        className: classNames('kbnPageTemplate__pageSideBar', rest.pageSideBarProps?.className),
+        ...pageSideBarProps,
+        className: classNames('kbnPageTemplate__pageSideBar', pageSideBarProps?.className),
```

Comment:
One other approach/solution could have been moving the `pageSideBarProps={{ ... }} ` declaration after `{...rest}` down below, but I figured destructuring out `pageSideBarProps` made sense since `bottomBarProps` was also already similarly destructured.

Reply:
I think there was a merge conflict with the collapsibility PR. This still needs to use the new variable containing the side bar classes. 
```suggestion
        className: classNames(sideBarClasses, pageSideBarProps?.className),
```

But at this point, with all the other stuff we do with this prop, I think it's good to pull it out of `...rest`. 👍

Reply:
Oh snap, thanks a ton for catching that Caroline! 🤦‍♀️